topics/squirrel/index.md

```diff
@@ -0,0 +1,12 @@
+---
+aliases: squirrel-lang
```

Comment:
It looks like the directory name and primary name of this topic should be `squirrel-lang` , because there is already a different project that is a popular packaging tool at github.com/squirrel , which has 90% of the repos currently under this topic. 

Alternately, you could make this file a disambiguation page that follows the pattern at https://github.com/topics/protocol  (similar to wikipedia) and add a curation for both `squirrel-lang` and `squirrel-installer`